`TestIgnorePuTTYSimpleChannel` flakiness

## Failure

#### Link(s) to logs

- https://console.cloud.google.com/cloud-build/builds;region=us-west1/a5b12c1c-8c14-49a8-a272-523f0089a4f0?project=ci-account

#### Relevant snippet

```
--- FAIL: TestIgnorePuTTYSimpleChannel (21.64s)
2022-07-12T10:43:05Z WARN [SESSION:N] Failed to complete stream. error:[
ERROR REPORT:
Original Error: *fs.PathError open /tmp/TestIgnorePuTTYSimpleChannel4274155353/002/log/upload/streaming/default/multi/dc87fa54-69c0-4de2-976e-7b5732f5b291/2712eaf9-e8fe-4f82-ae81-bfca7f1180d0/1.part: no such file or directory
Stack Trace:
	/workspace/lib/events/filesessions/filestream.go:181 github.com/gravitational/teleport/lib/events/filesessions.(*Handler).CompleteUpload
	/workspace/lib/events/stream.go:673 github.com/gravitational/teleport/lib/events.(*sliceWriter).completeStream
	/workspace/lib/events/stream.go:532 github.com/gravitational/teleport/lib/events.(*sliceWriter).receiveAndUpload
	/workspace/lib/events/stream.go:314 github.com/gravitational/teleport/lib/events.NewProtoStream.func1
	/opt/go/src/runtime/asm_amd64.s:1571 runtime.goexit
User Message: open /tmp/TestIgnorePuTTYSimpleChannel4274155353/002/log/upload/streaming/default/multi/dc87fa54-69c0-4de2-976e-7b5732f5b291/2712eaf9-e8fe-4f82-ae81-bfca7f1180d0/1.part: no such file or directory] events/auditwriter.go:546
```

--- a/lib/srv/regular/sshserver_test.go
+++ b/lib/srv/regular/sshserver_test.go
@@ -1909,7 +1909,7 @@ func TestX11ProxySupport(t *testing.T) {
 func TestIgnorePuTTYSimpleChannel(t *testing.T) {
 	t.Parallel()
 
-	f := newFixture(t)
+	f := newFixtureWithoutDiskBasedLogging(t)
 	ctx := context.Background()
 
 	listener, _ := mustListen(t)

